### PR TITLE
Periods control

### PR DESCRIPTION
**Pull request by butofleury**

### CODE REVIEW

**Comment by mestachs** [comment]
Can you explain what is this doing ?

File: package.json [hunk 1]
```diff
@@ -53,20 +53,24 @@
...
-    "rollup-plugin-postcss": "^2.0.3",
+    "rollup-plugin-postcss": "^1.6.2",
     "rollup-plugin-url": "^1.4.0"
   },
   "files": [
     "dist"
-  ]
+  ],
+  "dependencies": {
+    "formik": "^1.5.1",
```
Comment (mestachs) [inline package.json]: where is this used ?

**Comment by butofleury** [comment]
> Can you explain what is this doing ?

1. On burundi project, some invoices are generated only for the first month or the 2 last months of the quarter. So, we added a control on that to allow that kind of payment. 

2. We also changed the invoice buttons links component to allow viewing the invoices in the box Dialog. Because some org units are linked to 7 or 8 invoices. 

3. We added also the managment of category combo as org units, to allow display the donors in the org unit list.

File: package.json (hunk 1 above)
Comment (butofleury) [reply package.json]: sorry, I had not noticed that I added it in commit. I will delete it.

**Comment by mestachs** [comment]
I'm not a big fan of the button to select the factures.

it makes the invoices less discoverable/greppable for us (switching between projects is easer) and for clients (at least for first usage we don't hide info behind a button)

can't we trigger a behaviour where we show the previous format when not "too much invoices"
or show the 3 first invoices like before and a "more" button

IMO the label should be "Choisir une facture"

**Pull request merged**